### ISSUE TITLE
[TODO] oddessy stars

### ISSUE THREAD

**Issue by RVG-Revenge** [MaliciousFiles/EditPlayerData#41]
i set my games won to 1 million lol and my games played is stuck at a few hundred lmao it doesnt look right and theres no way to edit it.. also plz can u make it possible to edit the amount of oddessy stars u have

**Comment by MaliciousFiles** [comment]
Unfortunately, I can't do anything on games played. I scoured the NK backend, but I'm beginning to suspect that that's stored in Steam or something. You'll just have to select games won to be shown on your public profile instead of games played.

As for Odyssey stars, that I can probably do (though as always, no promises). I'll probably wait until I have a few other requests to do it though.

**Comment by MaliciousFiles** [comment]
Closing due to inactivity (also I think I tried and couldn't find it maybe).

**Comment by xdlr4562** [comment]
> Closing due to inactivity (also I think I tried and couldn't find it maybe).

I've kinda been wanting this to be added as well with the oddessy stars. If it isn't possible that's fine

**Comment by MaliciousFiles** [comment]
Fair enough. I'll take another look when I do the next wave of updates pretty soon.

**Comment by RVG-Revenge** [comment]
oh no way lol.. tysm :)
